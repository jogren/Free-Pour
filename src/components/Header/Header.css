header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 15px;
  height: 110px;
  background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('https://image.shutterstock.com/image-vector/vector-linear-illustration-bottles-glasses-260nw-667294063.jpg');
}

h1,
h2 {
  color: white;
  font-weight: 700;
}

h1 {
  font-size: 42px;
}

