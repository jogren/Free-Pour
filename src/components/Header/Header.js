import React from 'react';
import './Header.css';

const Header = () => {

  return(
    <header>
      <h1>Free Pour</h1>
      {/* <h2>Welcome, UserName</h2> */}
    </header>
  )

}

export default Header;