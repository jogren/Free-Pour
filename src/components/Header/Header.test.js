import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />)
  })

  it('should take a snapshot when passed the correct data', () => {
    expect(wrapper).toMatchSnapshot();
  });
})