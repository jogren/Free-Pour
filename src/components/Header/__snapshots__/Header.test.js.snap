// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should take a snapshot when passed the correct data 1`] = `
<header>
  <h1>
    Free Pour
  </h1>
  <h2>
    Welcome, UserName
  </h2>
</header>
`;
